add_users.py

```diff
@@ -0,0 +1,45 @@
+from app.db import get_session
+from app.models.users import Users
+from sqlmodel import select
+from app.auth.security import get_password_hash
+from logging import Logger
+
+def add_users(users_list_dict: dict):
+    """
+    This function creates new users in the spun up local database
+
+    Args:
+        users_list_dict: Should contain a list of dictionary 
+        usernames and plain text passwords for users
+
+    Raises:
+        Logger warning: If username already exists will raise a
+        warning and ignore adding this user.
```

Comment:
The logger warning is a side-effect it does not raise an exception.

This should either be removed, preferable, or "Raises" should be renamed to "Logs"